libs/server/Storage/Session/ObjectStore/SortedSetOps.cs

```diff
@@ -927,5 +927,194 @@ private GarnetStatus SortedSetDifference<TObjectContext>(ReadOnlySpan<ArgSlice>
 
             return GarnetStatus.OK;
         }
+
+        /// <summary>
+        /// Computes the intersection of multiple sorted sets and returns the members and their scores.
+        /// </summary>
+        /// <typeparam name="TObjectContext"></typeparam>
+        /// <param name="keys"></param>
+        /// <param name="pairs"></param>
+        /// <param name="objectContext"></param>
+        /// <returns></returns>
+        public GarnetStatus SortedSetIntersection<TObjectContext>(ReadOnlySpan<ArgSlice> keys, double[] weights, SortedSetAggregateType aggregateType, ref TObjectContext objectContext, out Dictionary<byte[], double> pairs)
```

Comment:
Method should be private as it's only used in this class, please move it to the bottom...